mitosheet/mitosheet/steps_manager.py

```diff
@@ -503,6 +503,9 @@ def handle_edit_event(self, edit_event: Dict[str, Any]) -> None:
         if self.curr_step_idx != len(self.steps_including_skipped) - 1:
             return
 
+        if edit_event.get('refresh_use_live_updating_hooks'):
+            self.update_event_count += 1
```

Comment:
Comments for sure!

Reply:
1. Why this exists (refreshData in useLiveUpdating hook for the toolbar etc)
2. Who it's used by
3. Other relevant potential solutions we considered, why they don't work -- why this is the easiest